README.md

```diff
@@ -1,38 +1,114 @@
 Minecraft Server Control Script
-=============================
+===============================
 
 A powerful command-line control script for UNIX and Linux powered Minecraft servers.
 
-
-## Index
-* [Features](#features)
+# Index
+* [Overview](#overview)
+* [Prerequisites for installation](#prerequisites-for-installation)
+  * [Required programs](#required-programs)
+  * [Configuring the firewall / NAT](#configuring-the-firewall--nat)
+  * [Mapping software (optional)](#mapping-software-optional)
 * [Installation](#installation)
-* [Usage](#usage)
-* [Server Customization](#server-customization)
+  * [Downloading the script](#downloading-the-script)
+  * [Configuration](#configuration)
+* [Creating and importing worlds](#creating-and-importing-worlds)
+  * [Create new world](#create-new-world)
+  * [Import existing world](#import-existing-world)
+* [Adjusting server options](#adjusting-server-options)
+  * [The mscs.properties file](#the-mscs.properties-file)
+  * [The msctl file](#the-msctl-file)
+* [Automated backups and restarts](#automated-backups-and-restarts)
+  * [Scheduling backups](#scheduling-backups)
+  * [Removing backups after X days](#removing-backups-after-x-days)
+  * [Scheduling restarts](#scheduling-restarts)
+* [Mapping the world](#mapping-the-world)
+* [Command reference](#command-reference)
+  * [Examples](#examples)
 * [License](LICENSE)
 * [Issues](#issues)
 
+## Overview
+**M**inecraft **S**erver **C**ontrol **S**cript (**MSCS**) is a server-management script for UNIX and Linux powered Minecraft servers. 
+Features include:
 
-## Features
 * Run multiple Minecraft worlds.
 * Start, stop, and restart single or multiple worlds.
 * Create, delete, disable, and enable worlds.
 * Includes support for additional server types: [Forge](http://www.minecraftforge.net/), [BungeeCord](http://www.spigotmc.org/wiki/bungeecord/), [SpigotMC](http://www.spigotmc.org/wiki/spigot/), etc.
 * Users automatically notified of important server events.
-* Uses the Minecraft [Query protocol](http://wiki.vg/Query) to keep track of current server conditions.
+* Uses the Minecraft [Query protocol](http:b //wiki.vg/Query) to keep track of current server conditions.
 * LSB and systemd compatible init script, allows for seamless integration with your server's startup and shutdown sequences.
 * Map worlds using the [Minecraft Overviewer](http://overviewer.org/) mapping software.
-* Backup worlds, and remove backups older than X days.
+* Automatically backup worlds, remove backups older than X days, and restart worlds.
 * Update the server and client software automatically.
 * Send commands to a world server from the command line.
 
-See the [Usage](#usage) section below for a description on how to use these features.
+Most of the general/beginner documentation is located in this file. For additional documentation that is not located in this file, please check the [wiki](https://github.com/Roflicide/MinecraftServerControlScript/wiki). 
```

Comment:
This should point to sandain's wiki `https://github.com/sandain/MinecraftServerControlScript/wiki`